app/database/models.py

```diff
@@ -58,6 +58,8 @@ class Event(Base):
     end = Column(DateTime, nullable=False)
```

Comment:
Remove the `.DS_Store` file and add it to the `.gitignore`

Reply:
It seems I can't delete the file already uploaded for some reason.
Can you help?

Reply:
`git rm .DS_Store`